src/view/errors/userReportableError.js

```diff
@@ -18,19 +18,14 @@ governing permissions and limitations under the License.
  */
 class UserReportableError extends Error {
   constructor(message, { originatingError, additionalInfoUrl } = {}) {
-    if (originatingError) {
-      // Log the originating error so that
-      // we can still see its stack trace.
-      console.error(originatingError);
-    }
-
```

Comment:
See monitorForOriginatingErrors for why I removed this. TL;DR: if this UserReportableError is handled, it's confusing to have the originating error logged to the console.